lib/liquid/register.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module Liquid
+  class Register
```

Comment:
What would this be used for?

Reply:
Nothing as of yet, this is future-proofing I guess, setting up the fact that most people copy and paste and if they copy another register at least its inheriting this. So if you have something like sorbet you can detect this class.

Reply:
:+1: